components/library/gnutls-3/manifests/sample-manifest.p5m

```diff
@@ -81,7 +81,7 @@ file path=usr/share/doc/gnutls/gnutls-logo.png
 file path=usr/share/doc/gnutls/gnutls-modauth.png
 file path=usr/share/doc/gnutls/gnutls-x509.png
 file path=usr/share/doc/gnutls/pkcs11-vision.png
-file path=usr/share/info/dir
+#file path=usr/share/info/dir
 file path=usr/share/info/gnutls-client-server-use-case.png
```

Comment:
Please don't edit the generated sample-manifest.p5m.
It's not only a template for creating the package manifest but also used to check if anything has changed for a new version or release. Only do the necessary changes to the real manifest.